### PR TITLE
updated adding EA to node to show TOML format

### PR DESCRIPTION
**Pull request by pappas999**
- Updated old JSON format of jobs using EAs to be new TOML format
- Other various changes to terms that have changed from JSON to TOML jobs
- Small grammatical changes
- Updated screenshots
- Removed section on 'excluding parameters' as I don't believe it applies with TOML

### CODE REVIEW

File: docs/External Adapters/node-operators.md [hunk 1]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
-  ]
-}
+Let's say you have an external adapter called `soccer-data` and it's registered in the `bridges` section. It takes 1 parameter: `playerId`, and in solidity, you'd pass the parameter with:
+```javascript
+request.add("playerId","12345678")
 ```
+How can you test the adapter on your node?
 
-Since `randomNumber` is not a [core adapter](../core-adapters/), it is an external adapter that each node on the request needs to have in order to fulfill the request. If you try to add a task type that does not exist already as a bridge, the job will fail to create.
+The easiest way is to setup a [Webhook Job](https://docs.chain.link/docs/jobs/types/webhook/) that uses the external adapter, and manually set the parameter.
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:55]:
```suggestion
The easiest way is to setup a [Webhook Job](/docs/jobs/types/webhook/) that uses the external adapter, and manually set the parameter.
```

File: docs/External Adapters/node-operators.md [hunk 2]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
 
-You can also exclude extra parameters from the spec when creating the job. This will allow requesters the ability to specify them per-request:
+> Note: You may need to set the [configuration variable](https://docs.chain.link/docs/configuration-variables/) `FEATURE_WEBHOOK_V2=true` in your `.env` file.
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:57]:
```suggestion
> Note: You might need to set the [configuration variable](/docs/configuration-variables/) `FEATURE_WEBHOOK_V2=true` in your `.env` file.
```

File: docs/External Adapters/node-operators.md [hunk 3]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
+
+To create a bridge on the node, go to the **Create Bridge** tab in the Node Operators Interface. Specify a name for the bridge, the URL for your external adapter, and optionally specify the minimum contract payment and number of confirmations for the bridge. Minimum contract payment is a fee paid in LINK for the Chainlink node making a call to the external adapter via the bridge. This fee is in addition to the fee specified at the global node level for processing job requests.
+
+![Node UI New Bridge Screen](/files/ea-new-bridge.png)
+
+The bridge name must be unique to the local node. The bridge URL is the URL of your external adapter, which can be local or on a separate machine.
+
+To add jobs that use the bridge, add a [Bridge Task](/docs/jobs/task-types/bridge/) to your job. The `bridge` task defined in the example below is defined as 'fetch' and the name of the bridge is `soccer-data`.
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:15]:
```suggestion
To add jobs that use the bridge, add a [Bridge Task](/docs/jobs/task-types/bridge/) to your job. The `bridge` task defined in the example below is defined as `fetch` and the name of the bridge is `soccer-data`.
```

File: docs/External Adapters/node-operators.md [hunk 4]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
-    { "type": "randomNumber" },
-    { "type": "copy",
-      "params": {"copyPath": ["details", "current"]}},
-    { "type": "multiply",
-      "params": {"times": 100 }},
-    { "type": "ethuint256" },
-    { "type": "ethtx" }
-  ]
-}
+Let's say you have an external adapter called `soccer-data` and it's registered in the `bridges` section. It takes 1 parameter: `playerId`, and in solidity, you'd pass the parameter with:
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:49]:
```suggestion
As an example, assume you have an external adapter named `soccer-data` that is registered in the `bridges` section and it takes one parameter named `playerId`. In solidity, you would pass the parameter with the following request:
```

File: docs/External Adapters/node-operators.md [hunk 5]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
+```javascript
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:50]:
```suggestion
```solidity
```

File: docs/External Adapters/node-operators.md [hunk 6]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
-  "tasks": [
-    { "type": "randomNumber" },
-    { "type": "copy" },
-    { "type": "multiply" },
-    { "type": "ethuint256" },
-    { "type": "ethtx" }
-  ]
-}
-```
+```toml
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:59]:
```suggestion
```jpv2
```
We're using custom highlighting for TOML and DOT now. See [https://github.com/smartcontractkit/documentation/commit/6b9e99d0d8092f49ab130536ecbfe9e267da27af](commit 6b9e99d0)

File: docs/External Adapters/node-operators.md [hunk 7]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
+
+To create a bridge on the node, go to the **Create Bridge** tab in the Node Operators Interface. Specify a name for the bridge, the URL for your external adapter, and optionally specify the minimum contract payment and number of confirmations for the bridge. Minimum contract payment is a fee paid in LINK for the Chainlink node making a call to the external adapter via the bridge. This fee is in addition to the fee specified at the global node level for processing job requests.
+
+![Node UI New Bridge Screen](/files/ea-new-bridge.png)
+
+The bridge name must be unique to the local node. The bridge URL is the URL of your external adapter, which can be local or on a separate machine.
+
+To add jobs that use the bridge, add a [Bridge Task](/docs/jobs/task-types/bridge/) to your job. The `bridge` task defined in the example below is defined as 'fetch' and the name of the bridge is `soccer-data`.
+
+```toml
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:17]:
```suggestion
```jpv2
```

File: docs/External Adapters/node-operators.md [hunk 8]
```diff
@@ -4,91 +4,76 @@ date: Last Modified
...
-Let's say you have an external adapter called `weather-cl-ea` and it's registered in the `bridges` tab. It takes 1 parameter: `city:boston`, and in solidity, you'd pass the parameter with: 
-```javascript
-request.add("city","boston")
+    fetch
+"""
 ```
-How can we test the adapter on our node?
 
-The easiest way is to setup a web job and manually add the parameter. 
+Adding the following into the TOML spec manually sets the parameters passed into the bridge task, and is equivalent to using `request.add` as shown above, or adding the data with the `--d` flag if you're using [curl](https://curl.se/).
```
Comment (dwightjl) [inline docs/External Adapters/node-operators.md:70]:
```suggestion
Adding the following into the TOML spec manually sets the parameters passed into the bridge task. It is equivalent to using `request.add` as shown above or adding the data with the `--d` flag if you're using [curl](https://curl.se/).
```

**Review by dwightjl — CHANGES_REQUESTED** [review]
Just a few more changes and this is ready to merge.
Remember to batch suggested edits together.

**Comment by pappas999** [comment]
thanks @dwightjl , suggested changes all LGTM

**Review by dwightjl — APPROVED** [review]

**Pull request merged**